Javascript/homework/responses/4.2-more-buttons.md

```diff
@@ -0,0 +1,17 @@
+### Week 4 - Toggling between digital and analog clocks and incoporating advanced features
+
+*For the final assignment, students will integrate all previous assignments into a single interface and add some bells and whistles.*
+
+#### Task 2: Adding radio buttons for clock preferences
+
+Next, we're going to create a set of radio buttons that allow a user to choose if they’d like to see the time on an analog clock or on a digital clock.
+
+First, open another form tag directly below the one you have for the timezone selector.  Similar to how you set up the timezone buttons, open a div tag with a unique class (I named mine "view") and insert 2 radio buttons. Make sure these radio buttons share the same `name` attribute and have unique `value` and `id` attributes.
+
+These two radio buttons should have the user input whether they'd like to see the time on a digital clock or an analog clock.  There's no need to add a submit button as we will set up our page to change on the click of the radio button rather than on the submit button.
+
+After this step, your page should look like this:
+
+![Screen Shot 2021-01-17 at 9.31.52 PM](/Users/danielleanzelone/Library/Application Support/typora-user-images/Screen Shot 2021-01-17 at 9.31.52 PM.png)
```

Comment:
Might want to fix the images. If you're using a local markdown editor, you'll need to upload the image to Github.